lib/export/export_sdf.rb

```diff
@@ -75,7 +74,18 @@ def filter_with_permission_and_detail_level(sample)
 
     def format_field(column, raw_value)
       field = column.gsub(/\s+/, '_').upcase
-      value = validate_value(raw_value)
+      reference_values = ['melting pt', 'boiling pt']
+      sample_column =
+        if reference_values.include?(column)
```

Comment:
maybe extract to own method with a cool name to make it clear that it converts the ruby range format in a more readable form?